testsuite/tests/lib-systhreads/boundscheck.ml

```diff
@@ -0,0 +1,39 @@
+(* TEST
+
+include systhreads
+* hassysthreads
+** not-windows
+*** bytecode
+*** native
```

Comment:
The test "correctly" fails on 4.14 with msvc64 and on mingw-w64 on trunk... and passes with the fix! IIUC although they're not real signals, the Windows code goes through the same paths with emulated signals.
```suggestion
** bytecode
** native
```